package.json

```
@@ -30,7 +30,7 @@
   },
   "peerDependencies": {
     "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0",
-    "typescript": "^3.7.5 || ^4.0.0"
+    "typescript": "^3.7.5 || ^4.0.0 || ^5.0.0 || ^5.0.0-beta"
```

I'm not keen on adding beta ranges to peer dependencies, let's just keep stable v5 and if someone wants to use beta channel it's on their own risk.
```suggestion
    "typescript": "^3.7.5 || ^4.0.0 || ^5.0.0"
```

---

Then it will not work. Currently there is no release version of v5, and no release date.

---

Sure, but you can still install with beta version using `force` flag, and it's good because user will know that there is no guarantee that this rule will work with beta TS version.
Alternatively we could add a beta range to peerDependencies and then release it under beta channel but not stable/latest.

---

If you would like to release this change in beta channel you can retarget this PR to the `next` branch.

---

`force` flag adds difficulties, especialy if you use some CI. All installation scripts must be rewriten with `force` flag. This is why I started this PR. I have retargeted to `next` branch.

---

Ok nice, now we can merge it as is, but for stable release we would need to add a regression test for TSv4.